### ISSUE TITLE
This package causes an error in LicensePage

### ISSUE THREAD

**Issue by granoeste** [fluttercommunity/flutter_launcher_icons#76]
LicensePage raises an error if this plugin is specified as dependencies

```
Caused by com.kiwi.fluttercrashlytics.FlutterException: 'package:flutter/src/foundation/licenses.dart': Failed assertion: line 159 pos 14: 'result.text.trimLeft() == result.text': is not true.
       at _AssertionError._doThrowNew(liberrors_patch.dart:40)
       at _AssertionError._throwNew(liberrors_patch.dart:36)
       at LicenseEntryWithLineBreaks.paragraphs.getParagraph(licenses.dart:159)
       at LicenseEntryWithLineBreaks.paragraphs(licenses.dart:203)
       at _SyncIterator.moveNext(libcore_patch.dart:151)
       at new List.from(libarray_patch.dart:40)
       at Iterable.toList(iterable.dart:398)
       at _LicensePageState._initLicenses.<fn>(about.dart:391)
       at _TaskEntry.run.<fn>(binding.dart:81)
       at Timeline.timeSync(timeline.dart:168)
       at _TaskEntry.run(binding.dart:78)
       at _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleEventLoopCallback(binding.dart:363)
       at _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._runTasks(binding.dart:341)
       at ._rootRun(zone.dart:1120)
       at _CustomZone.run(zone.dart:1021)
       at _CustomZone.runGuarded(zone.dart:923)
       at _CustomZone.bindCallbackGuarded.<fn>(zone.dart:963)
       at ._rootRun(zone.dart:1124)
       at _CustomZone.run(zone.dart:1021)
       at _CustomZone.bindCallback.<fn>(zone.dart:947)
       at Timer._createTimer.<fn>(libtimer_patch.dart:21)
       at _Timer._runTimers(libtimer_impl.dart:382)
       at _Timer._handleMessage(libtimer_impl.dart:416)
       at _RawReceivePortImpl._handleMessage(libisolate_patch.dart:171)
```

**Comment by MarkOSullivan94** [comment]
Hi thanks for reporting this. 

I'll need more information. Could you provide steps on how to replicate this issue?

**Comment by granoeste** [comment]
1. Define flutter_launcher_icons in pubspec.yaml

```
dev_dependencies:
  flutter_launcher_icons: ^0.7.0
```

2. Display LicensePage with the following code with onPressed of Button etc.
```
            showLicensePage(context: context);
```

[showLicensePage function \- material library \- Dart API](https://docs.flutter.io/flutter/material/showLicensePage.html)

I think the definition of [LICENSE](https://github.com/fluttercommunity/flutter_launcher_icons/blob/master/LICENSE#L3) is a problem...

**Comment by MarkOSullivan94** [comment]
Might have been due to the year / name not being put in, I've updated it in the master branch for now and should hopefully no longer cause any issues in the next release

**Comment by miguelcmedeiros** [comment]
@MarkOSullivan94 
This issue keeps happening, but not because of ```flutter_launcher_icons```, but because of ```image``` package. At least for me that's where it broke...

**Comment by MarkOSullivan94** [comment]
@granoeste v0.7.1 has been released - let me know if this fixes your issue

**Comment by micsan13br** [comment]
I am not sure about the status of this.

I've started working on a flutter project on and off around two months ago.
After noticing this week that showLicensePage generates an exception, I've disabled all packages from pubspec.yaml one at a time and found out that "flutter_launcher_icons" was responsible for the problem.

After that, I've updated the version to 0.7.3, uncommented the dependency and showLicensePage seems to still work fine.
